.github/workflows/release.yml

```diff
@@ -4,9 +4,6 @@ permissions:
   contents: read
 
 on:
-  pull_request:
```

Comment:
With this, the CI `generate changelog` can not pass `python hack/changelog.py *** refs/pull/1117/merge > /home/runner/work/changelog.txt` https://github.com/open-cluster-management-io/ocm/actions/runs/16792872396/job/47557752194?pr=1117

The release version is wrong(should be vx.x.x, but it is refs/pull/1117/merge) when running for PRs, so delete this on pull_request.